list/list.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Documentation</title>
+    <link rel="stylesheet" href="list.css">
+    <script src="https://kit.fontawesome.com/d5eef1e966.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="navbar">
```

Comment:
I think you should use more semantic tags here. Like `<header>` for heading, `<nav>` for navigation. It will increase the reachability of your code.